spec/requests/api/v1/users_spec.rb

```diff
@@ -55,5 +55,17 @@ def update_user_call(user_params)
         expect(response).to have_http_status(:unprocessable_entity)
       end
     end
+
+    describe 'DELETE /index' do
+      def delete_user_call(user_to_destroy)
+        delete api_v1_user_path(user_to_destroy)
+      end
+
+      context 'when is succesful' do
```

Comment:
You need to check here that the user is deleted also, please add a unit test for it.